source/PreferencesPanel.cpp

```diff
@@ -84,8 +93,15 @@ namespace {
 	// How many pages of controls and settings there are.
 	const int CONTROLS_PAGE_COUNT = 2;
 	const int SETTINGS_PAGE_COUNT = 2;
+
 	// Hovering a preference for this many frames activates the tooltip.
 	const int HOVER_TIME = 60;
+
+	// The url to the plugins-list.
+	const string PLUGIN_LIST_URL =
+		"https://raw.githubusercontent.com/endless-sky/endless-sky-plugins/master/generated/plugins.json";
+
+	const unsigned MAX_PLUGIN_INSTALLS_PER_PAGE = 18;
```

Comment:
why would it be signed? negative amounts?

Reply:
We generally just go with int unless we have a reason not to. It's not like you are really going to need that extra bit for the page count.

Though I believe you are using this for some size_t calculations, so maybe it could just be declared as that?

Reply:
using size_t